src/test/groovy/org/prebid/server/functional/model/request/activitie/Condition.groovy

```diff
@@ -0,0 +1,36 @@
+package org.prebid.server.functional.model.request.activitie
+
+import groovy.transform.EqualsAndHashCode
+import groovy.transform.ToString
+import org.hibernate.type.ComponentType
+
+@ToString(includeNames = true, ignoreNulls = true)
+@EqualsAndHashCode
+class Condition {
+
+    Component componentType
+    Component componentName
```

Comment:
Seem we aren't the same page, please take a look at the [doc](https://github.com/prebid/prebid-server/issues/2591#:~:text=value%20is%20utilized.-,Conditions,-In%20general%2C%20a), we use:

```
The general forms a condition take:

condition: {
    ATTRIBUTE1: [array of strings],
    ATTRIBUTE2: [array of strings]
}
```
And 
```
The following attributes may be seen:

componentName - this is the biddercode, the analytics adapter code, or the module code.
componentType - this is "bidder", "analytics", or for modules: if the module tag contains "rtd", then type is "rtd", otherwise type is "general".
```

Example: 
```
      condition: {
            componentType: ["bidder", "analytics"],
            componentName: ["bidderA"]
          },
```